### PR TITLE
Prep for 0.26.0 Release

### PR DESCRIPTION
**Pull request by Misty-W**
## Description

Prep for 0.26.0 Release.

---

### License

- [x] I license this contribution under the terms of the GNU GPL, version 3 and grant Unitary Fund the right to provide additional permissions as described in section 7 of the GNU GPL, version 3.

Before opening the PR, please ensure you have completed the following where appropriate.
- [x] I added unit tests for new code.
- [x] I used [type hints](https://www.python.org/dev/peps/pep-0484/) in function signatures.
- [x] I used [Google-style](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods) docstrings for functions.
- [x] I [updated the documentation](../blob/master/docs/CONTRIBUTING_DOCS.md) where relevant.
- [x] Added myself / the copyright holder to the AUTHORS file

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,7 +45,7 @@ Highlights from this release include a bug fixed in DDD, extended documentation
 - Calibration API simplifications (#1763) [@natestemen]
 - W state Benchmarking Circuit (#1723) [@purva-thakre]
 - Consider id as slack in ddd circuit mask (#1744) [@Aaron-Robertson]
-- Remaining Stuff related to Identity Insertion Scaling in Docs (#1759) [@purva-thakre]
+- Remaining Stuff related to Identity Insertion Scaling in Docs (#1759) [@natestemen]
```
Comment (natestemen) [inline CHANGELOG.md:48]: Not sure why this got changed, but this was definitely Purva's PR.

**Review by natestemen — COMMENTED** [review]
What a productive milestone!

I think it would be great to have a short code snippet to add to the release. It makes for a quick way to show people what might have changed, and it also makes for [good advertising material](https://twitter.com/unitaryfund/status/1643244993979588608?cxt=HHwWgIC9kbSn_c0tAAAA). Happy for you to choose what we should include (could be a single demo, or a few since there are multiple things I could imagine using).

File: CHANGELOG.md (hunk 1 above)
Comment (Misty-W) [reply CHANGELOG.md]: That's strange! Thanks for alerting me.

**Comment by Misty-W** [comment]
> I think it would be great to have a short code snippet to add to the release.

Good idea @natestemen! I assume you mean in the social posts and not embedded somehow in the release docs?

**Comment by natestemen** [comment]
I think in the release docs is the right place to put a short demo. Here's what I did for the last release: https://github.com/unitaryfund/mitiq/releases/tag/v0.25.0

**Comment by Misty-W** [comment]
Apologies for the force push during code review, I had forgotten to rebase after merging the PT PR 🥺.

**Review by natestemen — APPROVED** [review]
Thanks, and no worries about this force push!

If we could just add a disclaimer saying twirling hasn't been tested, so if any bugs or inconsistencies are encountered, [opening an issue](https://github.com/unitaryfund/mitiq/issues/new) would be much appreciated.

GTG for me after that!

**Pull request merged**